README.md

```diff
@@ -45,13 +17,11 @@ Rather than relying on multiple teams for tools like Intune or Jamf, build a uni
 
 4. Delete `.gitlab-ci.yml`, which is for GitLab CI/CD.
 
-5. Push your cloned and modified code to your repo.
-
-6. In GitHub, enable the `Apply latest configuration to Fleet` GitHub Actions workflow, and run workflow manually. Now, when anyone pushes a new commit to the default branch, the action will run and update Fleet. For pull requests, the workflow will do a dry run only.
+5. In GitHub, enable the `Apply latest configuration to Fleet` GitHub Actions workflow, and run workflow manually. Now, when anyone pushes a new commit to the default branch, the action will run and update Fleet. For pull requests, the workflow will do a dry run only.
 
-## Set up on GitLab
+## GitLab setup
 
-1. Create your own GitLab repository.
+1. Clone the [GitLab repository](https://gitlab.com/fleetdm/fleet-gitops), create your own GitLab repository, and push your clone to your new repo.
```

Comment:
At this point, the pipeline/action will run and fail because the secrets are not present. We could call it out.

Reply:
> he pipeline/action will run and fail because the secrets are not present. 

@getvictor only for GitLab right? I added a note.

Reply:
For both (although I haven't tried this updated flow with GitLab)